pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -637,14 +637,22 @@ func (r *reconcilerBase) setupOrTeardown(ctx context.Context, syncObj client.Obj
 	return nil
 }
 
+// ManagedByLabel is a uniform label that is applied to all resources which are
+// managed by reconciler-manager.
+func ManagedByLabel() map[string]string {
```

Comment:
do we need this func if it's only used once?

Reply:
I don't feel strongly, the original intent was to use this function for the label selector